tuxemon/graphics.py

```diff
@@ -271,7 +271,7 @@ def animation_frame_files(
 
     """
     frames = list()
-    pattern = re.compile(rf"{name}\.?[0-9]+\.png")
+    pattern = re.compile(rf"{name}\.?_[0-9]+\.png")
```

Comment:
There is another folder than uses this, so it will need to be _?

Reply:
Done!